inc/authldap.class.php

```diff
@@ -2802,8 +2802,10 @@ static function showUserImportForm(AuthLDAP $authldap) {
                   echo "<td>$label</td><td>";
                   $field_counter++;
                   echo "<input type='text' name='criterias[$field]' value='".
```

Comment:
this html tag is not closed and conflicts with https://github.com/glpi-project/glpi/pull/2635/files#diff-06e62a899e842968f6b16cb3b449bb6dR2808